sqlglot/expressions.py

```diff
@@ -2449,6 +2449,7 @@ class In(Predicate):
         "query": False,
         "unnest": False,
         "field": False,
+        "is_global": False,
```

Comment:
just call global

Reply:
I didn't call it global on purpose, because it's awkward when setting it as a kwarg due to it being a reserved keyword.

Reply:
i think there are other cases like that such as “as”

ok we’ll up to you